<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cronos</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="gpClock">
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="lbClock">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>27</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pbAdjustTime">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="gpChronometer">
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="lbChronometer">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>27</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pbStartChronometer">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pbClearChronometer">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pbPauseChronometer">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
